### ISSUE TITLE
Write a parameter fine-tuning script

### ISSUE THREAD

**Issue by kaiyamag** [kaiyamag/emotion-detection#8]
Or fine-tune through experimentation. The goal is to determine the highest-accuracy model configuration.

**Comment by kaiyamag** [comment]
Extensive testing of different model hyperparameter configurations. Experimental results are listed in my Jr. I.S. paper.
